README.md

```diff
@@ -18,90 +18,122 @@ Multiplex requires a JSON file with the following format:
         {
             "name": "common-params",
             "params":
+                [
+                    {
+                        "arg": "bs", "vals": [ "4k", "8k" ], "role": "client"
+                    },
+                    {
+                        "arg": "rw", "vals": [ "read", "write" ]
+                    }
+                ]
+        }
+    ],
+    "sets":
+        [
```

Comment:
Any particular reason why this is "unwound" more than the examples above?  I feel like we should be consistent...and the above is more compact.